web/src/components/molecules/ProjectOverview/ModelCard.tsx

```diff
@@ -1,28 +1,59 @@
 import styled from "@emotion/styled";
 
 import Card from "@reearth-cms/components/atoms/Card";
+import Dropdown from "@reearth-cms/components/atoms/Dropdown";
 import Icon from "@reearth-cms/components/atoms/Icon";
+import Menu from "@reearth-cms/components/atoms/Menu";
 
 export type Model = {
   id: string;
   name: string;
   description?: string;
+  key: string;
 };
 
 export type Props = {
   model: Model;
   onSchemaNavigate?: (modelId: string) => void;
   onContentNavigate?: (modelId: string) => void;
+  onModelDeletionModalOpen: (model: Model) => Promise<void>;
+  onModelUpdateModalOpen: (model: Model) => Promise<void>;
 };
 
-const ModelCard: React.FC<Props> = ({ model, onSchemaNavigate, onContentNavigate }) => {
+const ModelCard: React.FC<Props> = ({
+  model,
+  onSchemaNavigate,
+  onContentNavigate,
+  onModelDeletionModalOpen,
+  onModelUpdateModalOpen,
+}) => {
   const { Meta } = Card;
 
+  const ModelMenu = (
+    <Menu
+      items={[
+        {
+          key: "edit",
+          label: "Edit",
+          onClick: () => onModelUpdateModalOpen(model),
+        },
+        {
+          key: "delete",
+          label: "Delete",
```

Comment:
Both labels should be wrapped with i18n

Reply:
@KaWaite , wrapped the labels with `i18n`